client/src/utils/Link/CustomLink.tsx

```diff
@@ -0,0 +1,39 @@
+import { Link } from 'react-router-dom';
+import type { CustomLinkProps } from './CustomLink.types';
+
+const CustomLink: React.FC<CustomLinkProps> = ({ to, name, icon, linkClass, buttonClass, textClass }) => {
+    /* variable name starting with underscore is a convention
+    to indicate that the variable is going to be used inside the component. */
+
+    const _to = to || '/contactus'; // Default path if not provided
```

Comment:
Please don't use `Snake-case` constant variable declaration, make sure to use all capital letters ! 

```suggestion
    const TO = to || '/contactus'; 
    const NAME = name || undefined;
    const ICON = icon || undefined
```